.github/workflows/ci.yml

```diff
@@ -10,6 +10,8 @@ on:
 env:
   CYPRESS_KEY: ${{ secrets.CYPRESS_KEY }}
   NODE_VERSION: '16'
+  NX_BASE: origin/${{ github.event.pull_request.base.ref }}
+  NX_HEAD: origin/${{ github.event.pull_request.head.ref }}
```

Comment:
This flow is onlu triggered for the "push" event and not for PRs, should we remove those?

Reply:
true